scalableminds/webknossos
# Fix misc stuff when viewing tasks/annotations of another user

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- go to /users
- click "Show Annotations" for a user
- check that the annotations tab is activated initially
- check that the datasets tab is not visible
- switch tabs and ensure that everything looks alright

### Issues:
- fixes #6951

------
(Please delete unneeded items, merge only when none are left open)
